<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraConnectDialog</class>
 <widget class="QDialog" name="CameraConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to Camera</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Select mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="liveViewRadio">
          <property name="text">
           <string>LiveView</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">modeRadioGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QRadioButton" name="recordedRadio">
          <property name="text">
           <string>Recorded Video</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">modeRadioGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="imageRadio">
          <property name="text">
           <string>Image</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">modeRadioGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QDialogButtonBox" name="okCancelBox">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelBox</sender>
   <signal>accepted()</signal>
   <receiver>CameraConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelBox</sender>
   <signal>rejected()</signal>
   <receiver>CameraConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="modeRadioGroup"/>
 </buttongroups>
</ui>
